invoke/runners.py

```diff
@@ -476,7 +490,7 @@ def start(self, command):
             )
 
     def default_encoding(self):
-        return locale.getpreferredencoding(False)
+        return locale.getpreferredencoding()
```

Comment:
NOTE: We have issues only in Python 2.x on Linux and Mac OS when `uselocale` is `False`.

NOTE: Python 3 works fine on all platforms!